extensions/dacpac/package.nls.json

```diff
@@ -1,4 +1,6 @@
 {
 	"dacFx.settings": "Dacpac",
+	"dacFx.startTitle": "Data-tier Application Wizard",
+	"dacFx.category": "Data-tier Application",
```

Comment:
it's a little long having the command show up as "Data-tier Application: Data-tier Application Wizard." Should we just get rid of the category or shorten it to DacFx or Dacpac? @dzsquared for other ideas

Reply:
What about having the category be `DacFx`? That way it actually matches the extension name as well which makes more sense to me

Reply:
I would support the category being set to either "DacFx" or "SqlPackage"
unfortunately the extension name is "SQL Server Dacpac" -> this is fairly misleading on both the supported platforms and functionality.